rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2549,6 +2550,19 @@ def test_upgrade_db_31_to_32(user_data_dir):  # pylint: disable=unused-argument
     assert len([row[2] for row in result]) == 5
     assert len({row[2] for row in result}) == 5
 
+    cursor.execute('INSERT INTO ens_mappings(address, name) VALUES(?, ?)', ('0x45E6CA515E840A4e9E02A3062F99216951825eB2', 'LABEL1'))  # noqa: E501
+    try:
+        cursor.execute('INSERT INTO ens_mappings(address, name) VALUES(?, ?)', ('0x4362BBa5a26b07db048Bc2603f843E21Ac22D75E', 'LABEL1'))  # noqa: E501
+        raise AssertionError('Expected to get IntegrityError')
```

Comment:
That's not a test for the DB upgrade, but more for the table itself, right?

Also the way to test that pytest raises an exception is with this: https://docs.pytest.org/en/stable/reference/reference.html?highlight=raises#pytest.raises

Reply:
Thank for the link! Yeah, it's indeed more a db test. So I guess I should move it to the `test_db.py`, right ?